src/components/FieldLevelHelp/FieldLevelHelp.test.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-env jest */
+
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import FieldLevelHelp from './FieldLevelHelp';
+
+test('FieldLevelHelp renders properly', () => {
+  const component = renderer.create(
+    <FieldLevelHelp id="fieldlevelname1">Port Number</FieldLevelHelp>
+  );
+
+  let tree = component.toJSON();
```

Comment:
I assume you can use const here?

Reply:
Addressed in 2dd035d6034b21b780d5fc031b497172730c9348